Style guide buildout

tylerrzaneray opened the pull request:
Built out the Carhartt styleguide based on designs in Zeplin.

tylerrzaneray commented on README.md:
```diff
@@ -1,7 +1,7 @@
 DEMO: https://zaneray.github.io/carharrt-styleguide/
 
-# carharrt-styleguide
```
@cbaxter713 I noticed that this repo has a typo in the name... it's currently Carharrt (double R) but should be Carhartt (double T). I corrected any instances of the brand name within the project, but we should probably rename the repo after this branch is merged in.

cbaxter713 commented:
This is looking really good @tylerrzaneray , nice work!  Here a few things I am seeing that would be good to change:

- All clickable elements should have `cursor: pointer` (buttons, checkboxes, radios, inputs, accordions, selects, color selection, toggles,  etc)

- On the text inputs, don't have a transition on the outline, just the background color and text color. 
- Make the `.container` max-width at 1400px so the whole styleguide can show on a large screen.  No need to restrict it.  Use max-width and set width to 100%
- Remove font-size declaration on `.style-guide-container`.  This is effecting the rendering of natural font sizes for some things

tylerrzaneray commented:
Thanks for the feedback @cbaxter713. All the items you mentioned have been addressed. Additionally, I've removed the unlicensed versions of the "Conduit" font we were using and replaced them with a similar font from google fonts per a recommendation from Kirk.

There was no italic version of the google font and I didn't think it wise to add `font-style: italic;` everywhere we were using the google placeholder font, since this will degrade the way conduit looks when implemented on this style guide (there is a separate font face rule for conduit medium italic which doesn't require altering the font with the CSS property `font-style`)

cbaxter713 commented:
Sounds good man.  Looks good to me!
